src/transforms/log_to_metric.rs

```diff
@@ -120,26 +119,30 @@ impl LogToMetric {
 }
 
 enum TransformError {
-    FieldNotFound { field: Atom },
+    FieldNotFound {
+        field: String,
+    },
     TemplateParseError(TemplateError),
-    TemplateRenderError { missing_keys: Vec<String> },
-    ParseFloatError { field: Atom, error: ParseFloatError },
+    TemplateRenderError {
+        missing_keys: Vec<String>,
+    },
+    ParseFloatError {
+        field: String,
+        error: ParseFloatError,
+    },
 }
 
 fn render_template(s: &str, event: &Event) -> Result<String, TransformError> {
     let template = Template::try_from(s).map_err(TransformError::TemplateParseError)?;
     template.render_string(&event).map_err(|missing_keys| {
-        // convert to String to avoid printing Atom in Debug format
-        let missing_keys = missing_keys
-            .into_iter()
-            .map(|k| k.to_string())
-            .collect::<Vec<String>>();
+        // convert to String to avoid printing String in Debug format
+        let missing_keys = missing_keys.into_iter().collect::<Vec<String>>();
```

Comment:
Seems like this might have been left over? `missing_keys` is already a `Vec<String>`.